src/main/java/core/basesyntax/figure/figures/Circle.java

```diff
@@ -0,0 +1,45 @@
+package core.basesyntax.figure.figures;
+
+import core.basesyntax.color.Color;
+import core.basesyntax.figure.Figure;
+import core.basesyntax.figure.Shape;
+
+public class Circle extends Figure {
+
+    public static final Shape NAME = Shape.CIRCLE;
+    private int radius;
+
+    public Circle(Color color, int radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("     ##     ");
+        System.out.println("  #      #  ");
+        System.out.println(" #         #");
+        System.out.println("  #      #  ");
+        System.out.println("     ##     ");
+    }
+
+    @Override
+    public double getSquare() {
+        return (int) (Math.PI * radius * radius);
+    }
+
+    public int getRadius() {
+        return radius;
+    }
+
+    public void setRadius(int radius) {
+        this.radius = radius;
+    }
+
+    @Override
+    public String toString() {
+        return getColor() + " " + NAME
```

Comment:
```suggestion
        return getColor() + " " + getClass().getSimpleName()
```
there is no need to create enum

Reply:
I thought that enum isn't fitting there. Ok thanks

Reply:
we don't need to use enum here getClass().getSimpleName() is just a method